scripts/eslint-pre-commit

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
-
-git stash -q --keep-index
-
-# Test prospective commit
-git diff-index --cached HEAD --name-only --diff-filter ACMR | egrep '.js$' | xargs -I $(npm bin)/eslint {}
```

Comment:
Yeah, maybe. I wouldn't make it a blocker, since it's a minor thing, and I don't know how much time it takes to setup with auto-triggers and whatnot, but it seems _nice_ :cat: